package.json

```diff
@@ -41,6 +41,7 @@
   "dependencies": {
     "azure-functions-core-tools": "^3.0.3233",
     "dotenv": "^8.2.0",
+    "login": "^0.8.0",
```

Comment:
uninstall this, and `npm install migrate-mongo --save-dev` insteal

Reply:
After that, add these scripts into the package.json section like so from the article:
![image](https://user-images.githubusercontent.com/57023314/109436801-524f3780-79ef-11eb-998d-2e576a7f220d.png)

Reply:
Run the up command and check the database to see if your migration script has run properly! :)